pkgs/development/python-modules/omorfi-python/default.nix

```
@@ -0,0 +1,23 @@
+{ python3Packages
+, lib
```

```suggestion
{ buildPythonPackage
, lib
```
using python3Packages inside of python3Packages is strictly forbidden